src/routes/GDPRAgreement/container.js

```diff
@@ -44,8 +45,9 @@ export const GDPRConnector = ({ ref, ...props }) => (
 					color: customColor || color,
 					fontColor: customFontColor,
 					iconColor: customIconColor,
+					title: customTitle,
 				}}
-				title={I18n.t('GDPR')}
+				title={customTitle || I18n.t('GDPR')}
```

Comment:
Let's keep the other containers(other than Register, Chat and LeaveAMessage containers) with their own titles, okay?
Let's skip this one.

Reply:
:+1: